catalog/systemd.catalog.in

```diff
@@ -359,6 +359,13 @@ Support: %SUPPORT_URL%
 
 The unit @UNIT@ has successfully entered the 'dead' state.
 
+-- 0e4284a0caca4bfc81c0bb6786972673
+Subject: Unit skipped
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+The unit @UNIT@ was skipped and has entered the 'dead' state with result '@UNIT_RESULT@'.
```

Comment:
This should probably reference ExecCondition= somehow. i.e. say "An ExecCondition= command failed, and hence the unit was skipped." (I am not a native english speaker though, I am pretty sure this could be expressed better)